webflow-hacks/cms-hacks/3/3.js

```diff
@@ -0,0 +1,11 @@
+document.addEventListener('DOMContentLoaded', function () {
+  // get the amount (length) of ".cms-job-counter" classes
+  // and assign that amount to a constant named jobsCounter
+  var jobsCounter = document.querySelectorAll('[fs-hacks-element="CMS_JOBS_COUNTER"]').length;
+  console.log(jobsCounter);
```

Comment:
I think we should remove the log here.

Reply:
should be gone already! this was from a previous commit